src/AspNet.Security.OAuth.EVEOnline/EVEOnlineAuthenticationDefaults.cs

```diff
@@ -31,45 +31,41 @@ public static class EVEOnlineAuthenticationDefaults
     /// </summary>
     public static readonly string CallbackPath = "/signin-eveonline";
 
-    /// <summary>
-    /// Default values for the Tranquility (live) server.
-    /// </summary>
     public static class Tranquility
     {
         /// <summary>
         /// Default value for <see cref="OAuthOptions.AuthorizationEndpoint"/>.
         /// </summary>
-        public static readonly string AuthorizationEndpoint = "https://login.eveonline.com/oauth/authorize";
+        public static readonly string AuthorizationEndpoint = "https://login.eveonline.com/v2/oauth/authorize";
 
         /// <summary>
         /// Default value for <see cref="OAuthOptions.TokenEndpoint"/>.
         /// </summary>
-        public static readonly string TokenEndpoint = "https://login.eveonline.com/oauth/token";
+        public static readonly string TokenEndpoint = "https://login.eveonline.com/v2/oauth/token";
 
         /// <summary>
         /// Default value for <see cref="OAuthOptions.UserInformationEndpoint"/>.
         /// </summary>
+        [Obsolete("This value is no longer required for V2, information is now stored in the JWT.")]
```

Comment:
Using "V2" in the message might confuse people unfamiliar with the internals of the provider.

```suggestion
        [Obsolete("This endpoint is no longer used by the EVEOnline provider.")]
```